.env.template

```diff
@@ -1,5 +1,5 @@
 GITHUB_TOKEN = ""
-TIMEZONE_OFFSET = ""
+TIMEZONE ="America/Puerto_Rico"
```

Comment:
This is an interesting default! :P 

We should probably default to a major city, to maximize the chances that relatively few users will have to change it. I would suggest Los Angeles, to pick up the west coast, but could see an argument for New York or Chicago, or perhaps Tokyo. 

Or perhaps even better would be to default to Etc/UTC, as it is neutral, and would be a good choice for a global organization.

Reply:
got it!

Reply:
I added the changes, and noticed that If i don't add the commits specifically in VSCode, (this time I used the terminal) it wont add you as a collaborator.